### PR TITLE
Login

### PR DESCRIPTION
**Pull request by charliearlie**

### CODE REVIEW

File: src/components/Authentication/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+import { LOGGED_IN_USER, TOKEN } from '../../util/constants';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  inputRoot: {
+    'label + &': {
+      marginTop: theme.spacing(3),
+    },
+  },
+  input: {
+    borderRadius: 4,
```
Comment (LorenzoKoundouris) [inline src/components/Authentication/Login.tsx]: does this not need `px`?

File: src/components/Authentication/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+          placeholder="Enter a password"
+          type={showPassword ? 'text' : 'password'}
+          classes={{
+            root: classes.inputRoot,
+            input: classes.input,
+          }}
+          value={formData.password}
+          onChange={handleFormChange}
+          endAdornment={
+            <InputAdornment position="end" style={{ marginLeft: '0' }}>
```
Comment (LorenzoKoundouris) [inline src/components/Authentication/Login.tsx]: Can this style be moved out?

File: src/components/Authentication/Login.tsx (hunk 1 above)
Comment (charliearlie) [reply src/components/Authentication/Login.tsx]: The border radius is there 🤷‍♂

File: src/components/Authentication/Login.tsx (hunk 2 above)
Comment (charliearlie) [reply src/components/Authentication/Login.tsx]: Sadly no. Classes passed down to it were being overridden.

**Pull request merged**